src/llama.cpp

```diff
@@ -5907,6 +5912,7 @@ static void llm_load_print_meta(llama_model_loader & ml, llama_model & model) {
         LLAMA_LOG_INFO("%s: ssm_d_inner      = %u\n",     __func__, hparams.ssm_d_inner);
         LLAMA_LOG_INFO("%s: ssm_d_state      = %u\n",     __func__, hparams.ssm_d_state);
         LLAMA_LOG_INFO("%s: ssm_dt_rank      = %u\n",     __func__, hparams.ssm_dt_rank);
+        LLAMA_LOG_INFO("%s: ssm_dt_b_c_rms      = %d\n",     __func__, hparams.ssm_dt_b_c_rms);
```

Comment:
```suggestion
        LLAMA_LOG_INFO("%s: ssm_dt_b_c_rms   = %d\n",     __func__, hparams.ssm_dt_b_c_rms);
```

Since this string is printed along with the others, I think it should be aligned too :)

Otherwise it looks like this for Mamba-130m:

```
llm_load_print_meta: ssm_d_conv       = 4
llm_load_print_meta: ssm_d_inner      = 1536
llm_load_print_meta: ssm_d_state      = 16
llm_load_print_meta: ssm_dt_rank      = 48
llm_load_print_meta: ssm_dt_b_c_rms      = 0
llm_load_print_meta: model type       = 0.1B
```

Reply:
Great catch, thank you !